build-trust/ockam
# feat(rust): delete `TrustContext`

### DONE
1. Delete `TrustContext`. Replace with notion of Trusted Authority Identifier and CredentialRetriever
2. Remove CredentialsServer and related code
3. Use dedicated Members sqlite-backed storage for Authority node
4. Cache Credential in sqlite
5. Rework Credentials command
6. Migrations
7. Test migration
8. Test migration automatically
9. Support getting current time without std for ockam_node (migration storage) and ockam_identity (`now`)
10. Update Ockam App to use the new approach
11. Remove all mentions of trust context from the code
12. Update bats tests
13. Credential locking mechanism
14. Add online authority support outside of project
15. Improve authority node creation (checking if it's already running, fix logging)
16. Check enrollers can't create enrollers
17. Add additional identity service - identities verification
